test/IDE/complete_pound_expr.swift

```diff
@@ -15,26 +15,29 @@ func test1() {
   let _ = useSelector(##^POUND_EXPR_3^#)
 }
 
-// POUND_EXPR_INTCONTEXT: Begin completions, 5 items
+// POUND_EXPR_INTCONTEXT: Begin completions, 6 items
 // POUND_EXPR_INTCONTEXT-DAG: Keyword[#function]/None:            function[#String#]; name=function
 // POUND_EXPR_INTCONTEXT-DAG: Keyword[#file]/None:                file[#String#]; name=file
+// POUND_EXPR_INTCONTEXT-DAG: Keyword[#file]/None: filePath[#String#]; name=filePath
```

Comment:
@rintaro I don't really know what all of the fields in these lines mean. Is it correct to have `Keyword[#file]/None: filePath`, or does something need to be added to code completion so we get `Keyword[#filePath]` instead?

Reply:
We want `#filePath` here. You can make it by changing https://github.com/brentdax/swift/blob/0f0622d22b6d8e3327acc0ecff8967272342b838/lib/IDE/CodeCompletion.cpp#L3878
to `CodeCompletionKeywordKind::pound_filePath`.